brianlong/www.validators.app
# 179246833_add_avatar_url_to_validator_api_output

#### What's this PR do?
- Add avatar_url to api output

#### How should this be manually tested?
- Run localhost, send request from ie. Postman, check if you see `avatar_url` field

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/179246833

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
